### ISSUE TITLE
TensorFlow Lite Quantization Debugger Issue

### ISSUE THREAD

**Issue by Dnlee17** [tensorflow/tensorflow#58633]
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Ubuntu 18.04.3 
- TensorFlow library (version, if pip package or github SHA, if built from source):tensorflow-gpu==2.10.0

### 2. Code

def dense_block(inputs, filters):

    y = Dense(units=filters,use_bias=False)(inputs)
    y = BatchNormalization()(y)
    return y

def model():
    inputs = Input(shape = (112, 112, 3))
    .
    .
    .
    .
    .
    .
    .
    x = dense_block(x, 1)  -> (input_size=1,1,1,256, output_size=1,1,1,1)
    return Model(inputs, x)

### 3. conversion is successful,but the predicted value of int8 tflite has a large error value with the .pb weights file
If the conversion is successful, but the generated model is wrong, then state what is wrong:

- Model produces wrong results and accuracy drop 7~15%.

### 5. (optional) Any other info / logs

Question1:
My weights .pb file is successfully quantized into an int8 tflite model.
When doing tf.lite.experimental.QuantizationDebugger, the rmse/scale value of the last layer(Conv2D) is 2, which is far more than 0.289, but the rmse/scale values ​​in other layers are all It is around 0.289, and if I change the output of the last layer to more nodes, the value of rmse/scale of the last layer will be closer to 0.289.

Does anyone know what could be causing this to happen?
thanks!

**Comment by mohantym** [comment]
Hi @Dnlee17 !
Would it be possible to share complete stand alone code as colab gist to replicate this issue.
Thank you!
